Camunda.Api.Client/CamundaClient.cs

```diff
@@ -149,11 +149,22 @@ private void Initialize()
         {
             _httpMessageHandler = _httpMessageHandler ?? new ErrorMessageHandler();
 
-            _refitSettings = _refitSettings ?? new RefitSettings
+            _refitSettings = _refitSettings ?? GetRefitSettings(this._httpMessageHandler);
+        }
+
+        public static RefitSettings GetRefitSettings(HttpMessageHandler handler)
+        {
+            var refitSettings = GetRefitSettings();
+            refitSettings.HttpMessageHandlerFactory = () => handler;
+            return refitSettings;
+        }
+
+        public static RefitSettings GetRefitSettings()
```

Comment:
Feedback on my fork was to move this and the Jason setup from the static constructor to a proper factory class.  I lean towards minimal change on projects I don't have ownership on, but I'm willing to put in that work if the fork owners agree.